src/main/java/com/robinpowered/sdk/model/Busy.java

```diff
@@ -0,0 +1,50 @@
+package com.robinpowered.sdk.model;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+public class Busy {
```

Comment:
Missing class docblock, as well as doc blocks for all public members. Look at other models for some pasta